### DISCUSSION TITLE
Add a way to disable Infinty Nuke Block damage

### DISCUSSION THREAD

**Discussion by MojangPlsFix** [InnovativeOnlineIndustries/Industrial-Foregoing#1191] [Completed/Done]
It would be a nice feature to add a config value to disable the block damage of the Infinity Nuke. 
This would benefit Server Owners, so they don't have to ban them.

**Comment by Buuz135** [comment]
Claimed chunks disabled block damage

**Reply by MojangPlsFix** [reply]
Claiming is not a real solution, not every modpack will have something like FTB Chunks. 
One other problem is also the destruction of unclaimed chunks. It is nice, that players are not griefed, but if everything other than the claimed chunks are completely destructed, then this will scare of new players.
